hub/powertoys/awake.md

```diff
@@ -42,6 +47,11 @@ While PowerToys Awake can keep the computer awake indefinitely or temporarily, i
 
 This feature only works when PowerToys Awake is enabled and has one of the custom power states selected. It also does not prevent any user-initiated actions, such as manually putting the computer to sleep or hibernating it.
 
+## Lock screen behavior
+PowerToys Awake does not function when the lock screen is displayed. This is because the lock screen operates in a separate security context from the user session. When you lock your computer, Windows transitions to this secure context, and user-mode applications like PowerToys Awake cannot maintain their power requests.
```

Comment:
```suggestion
PowerToys Awake doesn't work when the lock screen is displayed. This limitation exists because the lock screen operates in a separate security context from the user session. When you lock your computer, Windows transitions to this secure context, and user-mode applications like PowerToys Awake can't maintain their power requests.
```